src/components/button/button.tsx

```diff
@@ -26,6 +26,7 @@ export const Button = React.forwardRef<HTMLButtonElement, ButtonProps>(
         disabled={props.disabled || props.state === 'loading'}
         aria-disabled={props.disabled || props.state === 'loading'}
         data-testid={testID}
+        aria-label={buttonProps?.['aria-label']}
         {...buttonProps}
```

Comment:
It's unnecessary to type `aria-label` explicitly, as it is passed with `{...buttonProps}`